### ISSUE TITLE
Enable playing EEs with the original's GUI

### ISSUE THREAD

**Issue by lynxlynxlynx** [gemrb/gemrb#2230]
Since the GUI is the main barrier, transplanting the original one would enable people to play EE mods. Roughly:
1. Install the two versions of the game
2. Mod the EE
3. Run a script or somesuch to copy over the original GUI files and marker files weidu uses for gametype detection
- are there any problematic naming collisions?
- do we need an extra game type, more checks?
- ???
4. Play the mashup

**Comment by lynxlynxlynx** [comment]
Maybe this is pointless, but we'll have to check all the main guis. If we can make them work instead, then why bother?

The inventory is already satisfactory enough (even without the extra overlay), maps work, **journal** just needs extra newlines added, **GUIREC** seems to mostly work (level up not tested), **spell books** are two sorts of broken, and the action bar works.

**Comment by lynxlynxlynx** [comment]
Ok, so now all the ingame windows are pretty much fully functional. :) It's clear some were replaced with lua versions that look different and there are some bits missing (eg. extra panel in the inventory), but it's surprisingly nothing critical.

On a different install I was able to go through bg1ee chargen, so at worst small things are broken there. This issue's idea is therefore pointless, we can just fix the remaining issues without worrying about lua for now.
